Add release notes for 2.8.0 (#8155)
PrefectHQ/prefect: RELEASE-NOTES.md

# Prefect Release Notes

## Release 2.8.0

### Flow run timeline view
![image](https://user-images.githubusercontent.com/6200442/212138540-78586356-89bc-4401-a700-b80b15a17020.png)

See https://github.com/PrefectHQ/prefect/pull/8153 for more details.

### Enhancements
- Add task option `refresh_cache` to update the cached data for a task run — https://github.com/PrefectHQ/prefect/pull/7856

> - Add logs when task run receives abort signal and is in non-final state — https://github.com/PrefectHQ/prefect/pull/8097

Reviewer: ```suggestion
- Add logs when a task run receives an abort signal and is in a non-final state — https://github.com/PrefectHQ/prefect/pull/8097
```

- Add [publishing of multiarchitecture Docker images](https://hub.docker.com/r/prefecthq/prefect-dev) for development builds  — https://github.com/PrefectHQ/prefect/pull/7900

### Fixes
- Add support for `allow_failure` to mapped task arguments — https://github.com/PrefectHQ/prefect/pull/8135
- Update conda requirement regex to support channel and build hashes — https://github.com/PrefectHQ/prefect/pull/8137
- Add numpy array support to orjson serialization — https://github.com/PrefectHQ/prefect/pull/7912

### Experimental
- Rename "Worker pools" to "Work pools" — https://github.com/PrefectHQ/prefect/pull/8107
- Rename default work pool queue — https://github.com/PrefectHQ/prefect/pull/8117
- Add worker configuration — https://github.com/PrefectHQ/prefect/pull/8100
- Add `BaseWorker` and `ProcessWorker` — https://github.com/PrefectHQ/prefect/pull/7996

> 

Reviewer: ```suggestion

### Documentation
- Add YouTube video to welcome page - https://github.com/PrefectHQ/prefect/pull/8090
- Add social links - https://github.com/PrefectHQ/prefect/pull/8088
- Increase visibility of Prefect Cloud and Orion REST API documentation - https://github.com/PrefectHQ/prefect/pull/8134
- Correct example of `infra_override` for deployments - https://github.com/PrefectHQ/prefect/pull/8101

```

Author: Can we exclude the 1 line changer 8101?

Author: ```suggestion

### Documentation
- Add YouTube video to welcome page - https://github.com/PrefectHQ/prefect/pull/8090
- Add social links - https://github.com/PrefectHQ/prefect/pull/8088
- Increase visibility of Prefect Cloud and Orion REST API documentation - https://github.com/PrefectHQ/prefect/pull/8134

```

# Prefect Release Notes

## Release 2.8.0

### Flow run timeline view

We're excited to announce that a new timeline graph has been added to the flow run page. 

> This view helps visualize how execution of your flow run takes place in time, opposed to the radar view that focuses on the structure of dependencies between task runs.

Reviewer: ```suggestion
This view helps visualize how execution of your flow run takes place in time, an alternative to the radar view that focuses on the structure of dependencies between task runs.
```

> ![image](https://user-images.githubusercontent.com/6200442/212138540-78586356-89bc-4401-a700-b80b15a17020.png)

Reviewer: ```suggestion
![The timeline view visualizes execution of your flow run over time](https://user-images.githubusercontent.com/6200442/212138540-78586356-89bc-4401-a700-b80b15a17020.png)
```
